README.md

```diff
@@ -0,0 +1,140 @@
+## SDK Features
+
+This repository contains snippets for many Temporal SDK features written in different Temporal SDK languages. This also
+contains a runner and language-specific harnesses to confirm feature behavior across versions.
+
+These SDK features serve several purposes:
+
+* Ensure parity across SDKs by having same-feature snippets adjacent to one another
+* Confirm feature behavior across SDK versions
+* Confirm history across SDK versions
+* Document features in different SDKs
+* Easy-to-use environment for writing quick workflows in all languages/versions
+
+## Building
+
+With latest [Go](https://golang.org/) installed, run:
+
+    go build
+
+## Running
+
+Prerequisites:
+
+* [Go](https://golang.org/) 1.17+
+* [JDK](https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot) 11+
+* [Node](https://nodejs.org) 16+
+
+Command:
+
+    sdk-features run --lang LANG [--version VERSION] [PATTERN...]
+
+Note, `go run .` can be used in place of `go build` + `sdk-features` to save on the build step.
+
+`LANG` can be `go`, `java`, or `ts`. `VERSION` is per SDK and if left off, uses the latest version set for the language
+in this repository.
+
+`PATTERN` must match either the features relative directory _or_ the relative directory + `/feature.<ext>` via
+[Go path match rules](https://pkg.go.dev/path#Match) which notably does not include recursive depth matching. If
+`PATTERN` arguments are not present, the default is to run all features.
+
+Several other options are available, some of which are described below. Run `sdk-features run --help` to see all
+options.
+
+**NOTE** There is [currently a bug](https://github.com/temporalio/temporal/issues/2207) with Temporalite that causes
+history checks to fail. Until it is fixed, the external server as shown below must be used.
+
+### External Server and Namespace
+
+By default, a [temporalite](https://github.com/DataDog/temporalite) is dynamically started at runtime to handle all
+feature runs. To not start the embedded server, use `--server` to specify the address of a server to use. Similarly, to
+not use a custom namespace that may not be registered on the external server, use `--namespace`
```

Comment:
To *not* use? What is the default namespace the tool uses?

Reply:
A dynamically named one that works with Temporalite

Reply:
Worth mentioning this here

Reply:
:+1: Will add

Reply:
Added